packages/teams-js/src/public/authentication.ts

```diff
@@ -40,6 +40,10 @@ export namespace authentication {
   /**
    * Initiates an authentication request, which opens a new window with the specified settings.
    *
+   * @remarks
+   * Be aware that the authentication flow must start and end on the same domain.
+   * If it doesn't, success and failure messages can't be returned to the window that initiated the call.
```

Comment:
For brevity, how about: The authentication flow must start and end from the same domain, otherwise success and failure messages won't be returned to the window that initiated the call.

Reply:
Sure, works for me. Thanks for the suggestion. Should be in the PR now.